readme.md

```diff
@@ -128,38 +130,38 @@ merge.all([x, y, z]) // => expected
 
 ## Options
 
-### `arrayMerge`
-deepmerge, by default, concatenates arrays and merges array values. 
-
-There are however nigh-infinite valid ways to merge arrays, and you may want to supply your own method. You can do this by passing an `arrayMerge` function as an option.
-
-The options object will include the default `isMergeableObject` implementation if the top-level consumer didn't pass a custom function in.
-
+### `arrayMerge: overwriteMerge`
```

Comment:
This isn't quite right - every section under the Options heading is a different option that you can pass to deepmerge.

Examples of different things you can pass to `arrayMerge` should be fourth level headers, underneath the `arrayMerge` heading.